Makefile

```diff
@@ -3,11 +3,11 @@ NATIVEARCH	 := $(shell go version | awk -F '[ /]' '{print $$5}')
 INTEGRATION  := nri-ecs
 BINARY_NAME   = $(INTEGRATION)
 GO_PKGS      := $(shell go list ./... | grep -v "/vendor/")
-RELEASE_VERSION := 1.0.0
+RELEASE_VERSION := 1.3.1
```

Comment:
Should we bump as well https://github.com/newrelic/nri-ecs/blob/92dd3d8ba815d3a93beb07453c3b1116fb60406a/pkg/ecs/integration.go#L13 ?

Reply:
not here , but for sure in the next "real" release of the integration

Reply:
even if we are running the 1.3.1? didn't we forgot it at some point?

Reply:
yes that needs to be updated in a new release of the integration, I will create an issue to just remove that constant and use ldflags for that variables.

Reply:
There is no need to bump it manually AFAIK, it should get automatically populated here: https://github.com/newrelic/nri-ecs/blob/92dd3d8ba815d3a93beb07453c3b1116fb60406a/.github/workflows/release.yml#L22

We could add a comment or change the default to a less meaningful value to clarify this.

Reply:
```suggestion
RELEASE_VERSION ?= "dev" # Populated by release workflow from git tag
```

Reply:
@roobre that works when releasing packaging , not when updating manifest, since like in this case we don't want to release a new version of the integration, we just want it to bump the bundle tag on the manifest

Reply:
https://github.com/newrelic/nri-ecs/blob/92dd3d8ba815d3a93beb07453c3b1116fb60406a/.github/workflows/release.yml#L22

Reply:
there will be no tag we we execute the manifest workflow manually.

Reply:
@roobre @paologallinaharbur  i updated the description of the PR . Maybe the intention is more clear now.

Reply:
I don't know why the comment with just the link got sent, I had more text written with that 😅 

I originally wanted to say that doing this automatically in one workflow but not in the other may confuse us at some point, making us think that the one that is not automated actually is, or vice-versa.

Cna you think about any way to overcome this? The only thing that comes to my mind is asking for the `RELEASE_VERSION` in the Github UI as a requirement for the manual workflow to run. There should be a way of doing this: https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow

Reply:
that looks good and make sense , we can ask it on the UI , I will implement it

Reply:
done

Reply:
not yet, havent implemented #40 , I would prefer to do that on an actual integration version upgrade, this is just manifest update of the bundle